service/src/test/java/bio/terra/catalog/controller/DatasetApiControllerTest.java

```diff
@@ -1,4 +1,4 @@
-package bio.terra.catalog.api;
+package bio.terra.catalog.controller;
```

Comment:
This is (sort of) an unrelated change. The openapi generated base classes are created in package `api`. It seems like we shouldn't put generated code in the same package as hand written code, so I moved our subclasses to `controller`.